processes/local_development/seed_test_data.py

```diff
@@ -15,24 +16,22 @@ def __init__(self, *args):
         self.test_data = {
             'title': 'test data 1',
             'uuid' : uuid4(),
-            'date_created' : datetime.now(timezone.utc).replace(tzinfo=None),
-            'date_modified' :datetime.now(timezone.utc).replace(tzinfo=None),
-            'frbr_status': 'to_do',
+            'frbr_status': 'complete',
             'cluster_status': False,
-            "source": 'test data source',
+            "source": TEST_SOURCE,
             'authors': ['Ayan||true'],
             'languages': ['Serbian'],
             'dates': ['1907-|publication_date'],
             'publisher': ['Project Gutenberg Literary Archive Foundation||'],
-            'identifiers': ['4064148285|owi','161468|oclc','161468|oclc','972173562|oclc','1467194159|oclc'],
+            'identifiers': [],
             'source_id': '4064148285|test',
             'contributors': ['Metropolitan Museum of Art (New York, N.Y.)|||contributor','Metropolitan Museum of Art (New York, N.Y.)|||repository','Thomas J. Watson Library|||provider'],
             'extent': ('11, 164 p. ;'),
             'is_part_of': ['Tauchnitz edition|Vol. 4560|volume'],
             'abstract': ['test abstract 1', 'test abstract 2'],
             'subjects': ['test subjects 1||'],
             'rights': ('hathitrust|public_domain|expiration of copyright term for non-US work with corporate author|Public Domain|2021-10-02 05:25:13'),
-            'has_part': ['1|example.com/1.pdf|met|text/html|{"catalog": false, "download": false, "reader": false, "embed": true}']
+            'has_part': ['1|example.com/1.pdf|{TEST_SOURCE}}|text/html|{"catalog": false, "download": false, "reader": false, "embed": true}']
```

Comment:
Oof - this one is a bit tricky cuz we need to use an f string but we have some double quotes. Lemme help out here.